pkg/command/devldap.go

```diff
@@ -0,0 +1,37 @@
+package command
+
+import (
+	"github.com/micro/cli"
+	"github.com/owncloud/ocis-devldap/pkg/command"
+	svcconfig "github.com/owncloud/ocis-devldap/pkg/config"
+	"github.com/owncloud/ocis-devldap/pkg/flagset"
+	"github.com/owncloud/ocis/pkg/config"
+	"github.com/owncloud/ocis/pkg/register"
+)
+
+// KonnectdCommand is the entrypoint for the konnectd command.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [comment on exported function DevLDAPCommand should be of the form "DevLDAPCommand ..."](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=4909758)

Reply:
Do you want to add kopano or devldap? ;-?